scripts/run_algo.py

```diff
@@ -1,26 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I think that for the 1.0 release it would be okay to just rename this, I imagine most people type this out each time they use it so it won't break much